### PR TITLE
Display sync status in favicon and title.

### PR DESCRIPTION
**Pull request by aviraldg**

### CODE REVIEW

File: src/components/structures/MatrixChat.js [hunk 1]
```diff
@@ -907,6 +907,11 @@ module.exports = React.createClass({
             console.warn("Failed to set badge count: "+e.message);
         }
         document.title = "Vector"+(notifCount > 0 ? " ["+notifCount+"]" : "");
+
+        if(state === "ERROR") {
+            this.favicon.badge("×");
```
Comment (ara4n) [inline src/components/structures/MatrixChat.js]: I think this has to be in a try block, as per the previous `this.favicon.badge()`?

File: src/components/structures/MatrixChat.js [hunk 2]
```diff
@@ -907,6 +907,11 @@ module.exports = React.createClass({
...
+            document.title = "Vector [ERROR]";
```
Comment (ara4n) [inline src/components/structures/MatrixChat.js]: I suggest [offline] rather than the scarier looking [ERROR] :)

**Comment by ara4n** [comment]
other than comments inline, this LGTM, thanks!

In future, if we get around to updating the badge count locally rather than basing it purely off the `/sync` results, then I suspect this should get backed out - as the unread count will still be meaningful and useful when offline (given the rest of Vector works fine offline)

**Comment by aviraldg** [comment]
@ara4n This should be ready to go; doesn't hide the badge count anymore, instead uses colour to indicate status.

**Comment by ara4n** [comment]
thanks! - just merging now. i'd prefer we stick with red badges for the count, but will switch to bright #f00 for the error state and #d00 for normal badge count.

**Pull request merged**